sduploader/sdstoreuploader.go

```diff
@@ -25,16 +31,33 @@ type sdStoreUploader struct {
 	buildID string
 	url     string
 	token   string
-	client  *http.Client
+	client  *retryablehttp.Client
 }
 
 // NewStoreUploader returns an SDUploader for a given build.
 func NewStoreUploader(buildID, url, token string) SDUploader {
+	// read config from env variables
+	if strings.TrimSpace(os.Getenv("LOGSERVICE_STOREAPI_TIMEOUT_SECS")) != "" {
```

Comment:
move it out of the function to where httpTimeout is defined at top ?
Maybe just call, LOGSERVICE_HTTP_TIMEOUT_SECS ?

Reply:
same as above